server/planning/content_profiles/profiles/event.py

```diff
@@ -43,7 +43,6 @@ class EventSchema(BaseSchema):
     reference = StringField()
     slugline = StringField()
     subject = subjectField
-    custom_vocabularies = schema.ListField()
```

Comment:
I would keep it in the schema for now so we can later do the migration, without it this data won't be returned from db